docs/how-to-guides/access-monitoring.md

```diff
@@ -0,0 +1,81 @@
+# Access Monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Prometheus](#prometheus)
+* [Alertmanager](#alertmanager)
+* [Grafana](#grafana)
+  * [Using Port Forward](#using-port-forward)
+  * [Using Ingress](#using-ingress)
+
+## Introduction
+
+This guide explains how to access the monitoring stack. Lokomotive ships monitoring stack in the form of prometheus-operator component. Learn more about the prometheus-operator component and its parameters [here](../configuration-reference/components/prometheus-operator.md).
+
+## Prerequisites
+
+- Deploy the prometheus-operator component in `monitoring` namespace. If different namespace used in the configuration then change the subsequent commands accordingly, `kubectl -n <namespace> ...`.
+
+## Prometheus
+
+Execute the command to port forward prometheus locally on port `9090`:
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-prometheus 9090:9090
```

Comment:
Nit: AFAIK you could specify `9090` instead of `9090:9090` with the same effect.